src/components/WithCozyIcon/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { Bold } from 'cozy-ui/transpiled/react/Text'
+import './styles.css'
+
+const WithCozyIcon = props => {
+  return (
+    <>
+      <Icon
+        icon="cloud"
+        size={16}
+        color="var(--charcoalGrey)"
+        className="CozyIcon"
+      />{' '}
+      <Bold tag="strong" {...props} />
+    </>
+  )
+}
```

Comment:
This component seems not very convenient since it can only show the cozy icon. I wonder if making it more generic makes sense.

I see the CozyIcon vertical align middle below, did you want to use a Media ?

Reply:
Related to https://github.com/cozy/cozy-passwords/pull/1#discussion_r359765703